src/probability/util_probScript.sml

```diff
@@ -1662,6 +1662,18 @@ val tail_countable = store_thm
  >- PROVE_TAC [COUNTABLE_IMAGE_NUM]
  >> RW_TAC std_ss [EXTENSION, IN_IMAGE, GSPECIFICATION]);
 
+Theorem THREE_SETS_INTER :
```

Comment:
This should be proved by `rw[EXTENSION] >> metis_tac[]`

Reply:
Thanks. I'll remove this useless lemma (turning it into a `by`) as it's only used once in `real_probabilityTheory`.